gcp/modules/couchdb-prometheus-exporter/main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "secrets_dir" {}
+variable "charts_dir" {}
+
+variable "replica_count" {}
+
+# Secret variables
+variable "secret_couchdb_admin_username" {}
+
```

Comment:
Just cosmetic, but maybe no blank line here?

Reply:
`terraform fmt` put it there. 🤷‍♀️

Reply:
:D ok then I guess